docs/guides/fractional-gates.ipynb

```diff
@@ -28,7 +28,7 @@
     "\n",
```

Comment:
<p>@kaelynj these outputs look quite a bit different? Is this ok?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2497/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Looking at the page, it says:</p><blockquote>These <em>fractional</em> gates are supported on <a href="https://qiskit.github.io/documentation/pr-2497/guides/processor-types#heron" target="_blank">Heron QPUs</a> in the form...</blockquote><p>But there are no open-access heron backends, only eagle. Does that mean this feature is only available to non-open-plan users?</p>

Reply:
<p>Ah yeah, then this feature isn't available to non-open plan users at this point. There we some Heron QPUs which were part of the open plan that have been retired.</p>